doc/src/build/rest-api.md

```diff
@@ -48,13 +48,62 @@ The `genesis` command creates Sui's initial state including four authorities and
 each with five gas objects:
 
 ```shell
-curl --location --request POST $SUI_GATEWAY_HOST/sui/genesis | json_pp
+curl --location --request POST $SUI_GATEWAY_HOST/sui/genesis \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "custom": false
+}' | json_pp
 ```
 
 These are Sui [objects](objects.md) used
 to pay for Sui [transactions](transactions.md#transaction-metadata),
 such as object transfers or smart contract (Move) calls.
 
+If you want to use a custom genesis then you can do the following:
+
+Add `genesis.conf` to the working directory of the rest server i.e. 
```

Comment:
This is the second place we talk about customizing genesis - the other one is in walllet.md which admittedly is mentioned below but I still feel like there is unnecessary redundancy here. Can we consolidate it a bit more (@Clay-Mysten?).

Reply:
Please link to the existing section instead and add anything new here there:
/build/wallet.md#customize-genesis